fence/agents/compute/fence_compute.py

```diff
@@ -347,8 +352,12 @@ def main():
 		elif options["--action"] in ["monitor", "status"]:
 			sys.exit(0)
 
-	# The first argument is the Nova client version
-	nova = nova_client.Client('2',
+	if LooseVersion(novaclient.__version__) <= LooseVersion('2.26.0') :
```

Comment:
Is that going to do anything useful if LooseVersion couldn't be imported?
Surely the agent would just throw an error and abort